core/src/main/java/io/aiven/klaw/service/TopicSyncControllerService.java

```diff
@@ -1076,7 +1076,7 @@ public ApiResponse updateSyncTopics(List<SyncTopicUpdates> updatedSyncTopics)
     if (listTopics.size() > 0) {
       try {
         return ApiResponse.builder()
-            .success(false)
+            .success(true)
             .message(manageDatabase.getHandleDbRequests().addToSynctopics(listTopics))
             .build();
```

Comment:
it might be worth sending an array back in the future? One for each of the topics synced along with the id of the topic etc so we can match them back in the UI when needed?